transifex/pull.py

```diff
@@ -19,225 +19,40 @@
 
     python transifex/pull.py git@github.com:edx/course-discovery.git --skip-compilemessages
 """
-import logging
-import os
-import re
-import subprocess
-import sys
-import time
 from argparse import ArgumentParser
-from contextlib import contextmanager
-from logging.config import dictConfig
 from os.path import abspath, dirname, join
-from urllib.parse import urlparse
 
 import yaml
 
 import concurrent.futures
-from github import Github, GithubException
+from utils.common import logger, repo_context
```

Comment:
Given that the script is run with:
```
$ python transifex/pull.py <URL>
```
I think you may want to import `from transifex.utils.common ...`. You may need that to get rid of `__init__.py`.